### ISSUE TITLE
Is it possible to inference the caption of one image without downloading the whole training dataset?

### ISSUE THREAD

**Issue by xden2331** [LuoweiZhou/VLP#29]
^

**Comment by LuoweiZhou** [comment]
@xden2331 Is this one image from COCO/flirckr30k or somewhere else? Also, please refer to my answer to this issue: https://github.com/LuoweiZhou/VLP/issues/30

**Comment by Hannah-Yao** [comment]
> @xden2331 Is this one image from COCO/flirckr30k or somewhere else? Also, please refer to my answer to this issue: #30

What if I want to inference captions for some images from neither COCO nor Flickr? Should I also need to do image feature extraction as mentioned in #30? And any different settings in execution script?
Would be much appreciated.

**Comment by Hannah-Yao** [comment]
What if I want to inference captions for some images outside of COCO/Flickr? Should I also need to do image feature extraction as mentioned in #30? Any different settings in the execution script?
Would be much appreciated.

**Comment by LuoweiZhou** [comment]
Yes, you will need to extract the feature on your own. Here is a reference: https://github.com/LuoweiZhou/detectron-vlp/issues/5
